dev-python/pyilmbase/pyilmbase-2.3.0.ebuild

```diff
@@ -0,0 +1,54 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PYTHON_COMPAT=( python2_7 )
+
+inherit autotools multilib-minimal python-single-r1 toolchain-funcs
+
+DESCRIPTION="ilmbase Python bindings"
+HOMEPAGE="http://www.openexr.com"
+SRC_URI="https://github.com/openexr/openexr/releases/download/v${PV}/${P}.tar.gz"
+LICENSE="BSD"
+
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="+numpy"
+
+REQUIRED_USE="${PYTHON_REQUIRED_USE}"
+
+RDEPEND="${PYTHON_DEP}
+	>=dev-libs/boost-1.62.0-r1[${MULTILIB_USEDEP},python(+),${PYTHON_USEDEP}]
+	~media-libs/ilmbase-${PV}:=[${MULTILIB_USEDEP}]
+	numpy? ( >=dev-python/numpy-1.10.4 )"
+DEPEND="${RDEPEND}
+	${PYTHON_DEP}
+	>=virtual/pkgconfig-0-r1[${MULTILIB_USEDEP}]"
+
+AT_M4DIR=m4
```

Comment:
But you aren't using it, are you?

Reply:
Not sure, but I think the reason it's been included in previous ebuilds is because the .pc file is generated by pkg-config?

Reply:
More likely it was needed for `eautoreconf` but I don't see `eautoreconf` here.

Reply:
Ah. I was finally able to remove eautoreconf!